service/app/controllers/certmanagement/CertificateController.java

```diff
@@ -0,0 +1,20 @@
+package controllers.certmanagement;
+
+import controllers.BaseController;
+import org.sunbird.common.models.util.ActorOperations;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.request.UserRequestValidator;
+import play.libs.F;
+import play.mvc.Result;
+
+public class CertificateController extends BaseController {
```

Comment:
java doc required for class and all public method.

Reply:
Done